llvm/include/llvm/CodeGen/TargetLowering.h

```diff
@@ -242,11 +242,15 @@ class LLVM_ABI TargetLoweringBase {
 
   /// Enum that describes what type of support for selects the target has.
   enum SelectSupportKind {
```

Comment:
I'm not sure it make sense to duplicate these entries. This is an optimization hint for what the ISA looks like for a select. ctselect support would be a separate question, not an expanded set of answers